playbooks/kafka_broker.yml

```diff
@@ -55,7 +67,7 @@
     - name: Proceed Prompt
       pause:
         prompt: "Press Enter to Proceed to Next Node. Ctrl + C to Abort"
-      when: kafka_broker_pause_rolling_deployment|bool
+      when: kafka_broker_pause_rolling_deployment|bool and not kraft_enabled|bool
```

Comment:
Didn't get the idea of having the condition here

Reply:
thanks, removed it as it was redundant